conf/tachyon-env.sh.template

```diff
@@ -29,19 +29,18 @@
 # Configuration Examples:
 #
 # Environment variables for service bind
-# TACHYON_MASTER_BIND_HOST=`hostname -A | cut -d" " -f1`
-# TACHYON_MASTER_WEB_BIND_HOST=0.0.0.0
-# TACHYON_WORKER_BIND_HOST=`hostname -A | cut -d" " -f1`
-# TACHYON_WORKER_DATA_BIND_HOST=`hostname -A | cut -d" " -f2`
-# TACHYON_WORKER_WEB_BIND_HOST=0.0.0.0
+# export TACHYON_MASTER_BIND_HOST=$(hostname -A | cut -d" " -f1)
```

Comment:
are these exports absolutely necessary?  if tachyon-env.sh is just sourced by other files, we shouldn't pollute global environment if it is possible.